### DISCUSSION TITLE
Is it possible to use the plugin to start a Foreground Service?

### DISCUSSION THREAD

**Discussion by ComptonAlvaro** [thudugala/Plugin.LocalNotification#415] [Q&A]
To start a foreground service, I need to use notification. I will call this standard notification, to diferentiate from the notifications from this plugin. The code is this:

```
_notificationManager = (GetSystemService(Context.NotificationService) as NotificationManager)!;

createNotificationChannel(_notificationManager);


Notification notification = new Notification.Builder(this, NOTIFICATION_CHANNEL_ID)
    .SetContentTitle("My Title")
    .SetContentText($"{DateTime.Now}.")
    .SetSmallIcon(Resource.Drawable.logo32x32)
    .SetAutoCancel(false)
    .SetOngoing(false)
    .Build();

StartForeground(NOTIFICATION_ID, notification);
```

The problem is that if I use the standar notification and later I want to use this plugin, I have two notifications in the status bar. The first one is the standar notification that I can't close and the second one is the notification of this plugin.

If I use always the standard notification, I have only only one notification in the status bar, so for me is better because I don't use a lot of space from the notifications place.

So in sumary, I would like to know if it is possible to use this plugin to start the foreground service and to have only only one notification in the status bar.

Thanks.

**Comment by thudugala** [comment]
@ComptonAlvaro At the moment, the plugin does not support creating foreground service.
